src/test/ui/issues/issue-2214.rs

```diff
@@ -23,7 +23,7 @@ fn lgamma(n: c_double, value: &mut isize) -> c_double {
 mod m {
     use libc::{c_double, c_int};
 
-    #[link_name = "m"]
+    #[link(name = "m")]
```

Comment:
fail to link to libm on msvc x86: https://github.com/rust-lang-ci/rust/runs/1083564279#step:23:17053

Reply:
Anyone know why?  Should it be looking for a dll instead?

Reply:
I'm pretty sure msvc does not have libm (it is part of the universal C runtime). I think this attribute can be removed.  I think on all platforms the `lgamma` function is available through one way or another (primarily via the `libc` crate I believe, which handles linking `-lm` as appropriate). Considering this test has been working with a bogus attribute, I think it should be fine to remove.